grep.c

```diff
@@ -16,6 +16,22 @@ static int grep_source_is_binary(struct grep_source *gs,
 
 static struct grep_opt grep_defaults;
 
+#ifdef USE_LIBPCRE2
+static pcre2_general_context *pcre2_global_context;
+
+#ifdef USE_NED_ALLOCATOR
```

Comment:
Please do drop any mention of `nedmalloc` in this PR. There is nothing specific to `nedmalloc` about overriding PCRE2's allocator with whatever Git uses, whether the allocator was overridden at build time or at runtime does not matter one bit.